### PR TITLE
Added Edit Article Page.

### PR DESCRIPTION
**Pull request by mayurkapadia35**
--- Added edit-article page
--- Putted EditArticle Button on single article page for admin, superAdmin & editors
--- Set the css for the button.
--- Add a dynamic title for the page.

### CODE REVIEW

**Comment by ldeveloper7** [comment]
Give Proper Title & Description to Every PR

File: src/components/singlearticle/singlearticle.html [hunk 1]
```diff
@@ -14,6 +14,7 @@
                 {{cat.name}}
               </v-btn>
             </span>
+              <v-btn outline color="blue" v-if="editarticleflag" :to="'/editpost/'+singlearticle.slug" class="ml-5">Edit Article</v-btn>
```
Comment (ldeveloper7) [inline src/components/singlearticle/singlearticle.html]: Give this `float: right;` css

Comment (mayurkapadia35) [reply src/components/singlearticle/singlearticle.html]: :+1:

File: src/components/edit-post/editpost.html [hunk 2]
```diff
@@ -0,0 +1,194 @@
+<div>
+  <headermenu/>
```
Comment (ldeveloper7) [inline src/components/edit-post/editpost.html]: Add dynamic title here

Comment (mayurkapadia35) [reply src/components/edit-post/editpost.html]: I didn't getting you, can you elaborate?

Comment (ldeveloper7) [reply src/components/edit-post/editpost.html]: what is this PR for ??
https://github.com/ldeveloper7/vue-start/pull/44

Comment (mayurkapadia35) [reply src/components/edit-post/editpost.html]: That PR is generated by other developer

Comment (ldeveloper7) [reply src/components/edit-post/editpost.html]: So ??

Comment (ldeveloper7) [reply src/components/edit-post/editpost.html]: is it not part of your project ??

Comment (ldeveloper7) [reply src/components/edit-post/editpost.html]: Are you not checking the code of other developer ??

Comment (ldeveloper7) [reply src/components/edit-post/editpost.html]: Are you not checking, what the new thing is done by other developer ??

Comment (ldeveloper7) [reply src/components/edit-post/editpost.html]: if other developer has implemented the functionality of dynamic title as per page,
And you are adding new page, than you must have to add that dynamic title thing for your page..

Comment (ldeveloper7) [reply src/components/edit-post/editpost.html]: if you are working with team, than you must have to track what the things are going in your project

Comment (ldeveloper7) [reply src/components/edit-post/editpost.html]: Now just set dynamic title for your edit article page...
Make it fast..

Comment (mayurkapadia35) [reply src/components/edit-post/editpost.html]: Ok

Comment (mayurkapadia35) [reply src/components/edit-post/editpost.html]: :+1:

Comment (ldeveloper7) [reply src/components/edit-post/editpost.html]: Your branch is not update with master branch

Comment (mayurkapadia35) [reply src/components/edit-post/editpost.html]: :+1:

**Pull request merged**